python/lsst/qa/explorer/writeParquet.py

```diff
@@ -0,0 +1,107 @@
+from lsst.daf.persistence.butler import Butler
+from lsst.pex.config import (Config, Field, ConfigField, ListField, DictField, ConfigDictField,
+                             ConfigurableField)
+from lsst.pipe.base import Task, CmdLineTask, ArgumentParser, TaskRunner, TaskError
+from lsst.coadd.utils import TractDataIdContainer
+from lsst.pipe.tasks.multiBand import MergeSourcesTask, MergeSourcesConfig
+from lsst.pipe.tasks.multiBand import _makeGetSchemaCatalogs
+from lsst.coadd.utils.coaddDataIdContainer import ExistingCoaddDataIdContainer
+
+
+import functools
+import re
+import pandas as pd
+
+from .table import ParquetTable
+
+
+class WriteObjectTableConfig(MergeSourcesConfig):
+    priorityList = ListField(dtype=str, default=['HSC-G', 'HSC-R', 'HSC-I', 'HSC-Z', 'HSC-Y'],
+                             doc="Priority-ordered list of bands for the merge.")    
+    engine = Field(dtype=str, default="pyarrow", doc="Parquet engine for writing (pyarrow or fastparquet)")
+
+class WriteObjectTableTask(MergeSourcesTask):
+    """Write filter-merged source tables to parquet
+    """
+    _DefaultName = "writeObjectTable"
+    ConfigClass = WriteObjectTableConfig
+
+    inputDatasets = ('forced_src', 'meas', 'ref')
+    outputDataset = 'obj'
+
+    # Hack, since we're not using this...
+    getSchemaCatalogs = lambda x : {}
+
+    @classmethod
+    def _makeArgumentParser(cls):
+        """Create a suitable ArgumentParser.
+
+        We will use the ArgumentParser to get a provide a list of data
```

Comment:
redundancy? "...to get a provide a list..."

Reply:
Ha! I just copied from https://github.com/lsst/pipe_tasks/blob/cf273cdbbb95f59b14fd3f4e6cddfab8dfd9637f/python/lsst/pipe/tasks/multiBand.py#L432 without reading carefully!